### PR TITLE
Azure Architecture Center - Style Formatting

### PR DESCRIPTION
**Pull request by ckittel**
This PR is to:
- do a usability pass to the deployment instructions and related files in the accelerator.
- Address Azure AD -> Entra rebranding
- Add some TODOs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,515 +1,479 @@
...
+             "type": "Role"
+           },
+           {
+             "id": "$pimRole",
+             "type": "Role"
+           }
+         ]
+       }
+     ]
+   }"@
```
Comment (sebassem) [inline README.md:140]:
```suggestion
   }
"@

```

File: README.md [hunk 2]
```diff
@@ -1,515 +1,479 @@
...
+   ```powershell
+   $parameters4AzAPICallModule = @{
+     #SubscriptionId4AzContext = $null #specify Subscription Id
+     #DebugAzAPICall = $true
+     #WriteMethod = 'Output' #Debug, Error, Host, Information, Output, Progress, Verbose, Warning (default: host)
+     #DebugWriteMethod = 'Warning' #Debug, Error, Host, Information, Output, Progress, Verbose, Warning (default: host)
+     #SkipAzContextSubscriptionValidation = $true #Use if the account doesn´t have any permissions on Management Groups,  Subscriptions, Resource Groups or Resources
+   }
+ 
+   $azAPICallConf = initAzAPICall @parameters4AzAPICallModule
```
Comment (sebassem) [inline README.md:79]:
```suggestion
$azAPICallConf = initAzAPICall @parameters4AzAPICallModule

```

File: README.md [hunk 3]
```diff
@@ -1,515 +1,479 @@
...
+       }
+     ]
+   }"@
+ 
+   $AzGovVizAppObjectId = (AzAPICall -method POST -body $body -uri "$($azAPICallConf['azAPIEndpointUrls'].MicrosoftGraph)/ v1.0/applications" -AzAPICallConfiguration $azAPICallConf -listenOn 'Content' -consistencyLevel 'eventual').id
+ 
+   do {
+     Write-Host "Waiting for the AzGovViz service principal to get created..."
+     Start-Sleep -seconds 20
+     $AzGovVizAppId = (AzAPICall -method GET -uri "$($azAPICallConf['azAPIEndpointUrls'].MicrosoftGraph)/v1.0/       applications/$AzGovVizAppObjectId" -AzAPICallConfiguration $azAPICallConf -listenOn 'Content' -consistencyLevel  'eventual' -skipOnErrorCode 404).appId
```
Comment (sebassem) [inline README.md:147]:
```suggestion
     $AzGovVizAppId = (AzAPICall -method GET -uri "$($azAPICallConf['azAPIEndpointUrls'].MicrosoftGraph)/v1.0/applications/$AzGovVizAppObjectId" -AzAPICallConfiguration $azAPICallConf -listenOn 'Content' -consistencyLevel  'eventual' -skipOnErrorCode 404).appId

```

File: README.md [hunk 4]
```diff
@@ -1,515 +1,479 @@
...
+           {
+             "id": "$pimRole",
+             "type": "Role"
+           }
+         ]
+       }
+     ]
+   }"@
+ 
+   $AzGovVizAppObjectId = (AzAPICall -method POST -body $body -uri "$($azAPICallConf['azAPIEndpointUrls'].MicrosoftGraph)/ v1.0/applications" -AzAPICallConfiguration $azAPICallConf -listenOn 'Content' -consistencyLevel 'eventual').id
```
Comment (sebassem) [inline README.md:142]:
```suggestion
   $AzGovVizAppObjectId = (AzAPICall -method POST -body $body -uri "$($azAPICallConf['azAPIEndpointUrls'].MicrosoftGraph)/v1.0/applications" -AzAPICallConfiguration $azAPICallConf -listenOn 'Content' -consistencyLevel 'eventual').id

```

File: README.md [hunk 5]
```diff
@@ -1,515 +1,479 @@
...
-    "subject":"$subject",
-    "issuer":"https://token.actions.githubusercontent.com",
-    "name":"AzGovVizCreds"
-    }
-"@
+  ],
+  "subject":"$subject",
+  "issuer":"https://token.actions.githubusercontent.com",
+  "name":"AzGovVizCreds"
+}"@
```
Comment (sebassem) [inline README.md:226]:
```suggestion
}
"@

```

File: README.md [hunk 6]
```diff
@@ -1,515 +1,479 @@
...
-    {
-    "enableIdTokenIssuance": true
+    "implicitGrantSettings": {
+      "enableIdTokenIssuance": true
     }
-    }
-    }
-"@
+  }
+}"@
```
Comment (sebassem) [inline README.md:288]:
```suggestion
}
"@

```

File: README.md [hunk 7]
```diff
@@ -1,515 +1,479 @@
...
+      {
+        "value": "user_impersonation",
+        "adminConsentDescription": "AzGovViz Web App Microsoft Entra ID authentication",
+        "adminConsentDisplayName": "AzGovViz Web App Microsoft Entra ID authentication",
+        "type": "User",
+        "id": "$webAppSPAppId"
+      }
+    ]
+  }
+}"@
```
Comment (sebassem) [inline README.md:319]:
```suggestion
}
"@

```

File: README.md [hunk 8]
```diff
@@ -1,515 +1,479 @@
...
+# Generate client secret
+$body = @"
+{
+  "passwordCredential":{
     "displayName": "AzGovVizWebAppSecret"
-    }
-    }
-"@
+  }
+}"@
```
Comment (sebassem) [inline README.md:329]:
```suggestion
}
"@

```

**Review by sebassem — CHANGES_REQUESTED** [review]
A couple of bugs due to spacing

File: README.md [hunk 9]
```diff
@@ -1,515 +1,479 @@
...
+
+   ```powershell
+   $parameters4AzAPICallModule = @{
+     #SubscriptionId4AzContext = $null #specify Subscription Id
+     #DebugAzAPICall = $true
+     #WriteMethod = 'Output' #Debug, Error, Host, Information, Output, Progress, Verbose, Warning (default: host)
+     #DebugWriteMethod = 'Warning' #Debug, Error, Host, Information, Output, Progress, Verbose, Warning (default: host)
+     #SkipAzContextSubscriptionValidation = $true #Use if the account doesn´t have any permissions on Management Groups,  Subscriptions, Resource Groups or Resources
+   }
+
```
Comment (ckittel) [inline README.md:78]:
```suggestion

```

**Review by sebassem — APPROVED** [review]
LGTM

**Pull request merged**